File: lab03/Pessoa.cpp
#include <iostream>
#include "Pessoa.hpp"

using namespace std;

    Pessoa::Pessoa(){}

    Pessoa::Pessoa(string nome, string endereco){
        this->nome = nome;
        this->endereco = endereco;
    }

    string Pessoa::getNome(){
        return nome;
    }

    string Pessoa::getEndereco(){
        return endereco;
    }

    void Pessoa::setNome(string nome){
        this->nome = nome;
    }

    void Pessoa::setEndereco(string endereco){
        this->endereco = endereco;
    }

File: lab07/Pessoa.java
public class Pessoa{
    private String nome;
    private String cpf;

    public Pessoa (String nome, String cpf){
        this.nome = nome;
        this.cpf = cpf;
    }

    public String getNome(){
        return nome;
    }

    public String getCpf(){
        return cpf;
    }

    public void setNome(String nome){
        this.nome = nome;
    }

    public void setCpf(String cpf){
        this.cpf = cpf;
    }

    @Override
    public String toString(){
        return "Nome: " + nome + ", CPF: " + cpf;
    }
}